app/templates/components/lg-image-set-editor.hbs

```diff
@@ -0,0 +1,34 @@
+<div class='col-sm-2'>
+  {{#if imageSet.mainImage}}
+    <img {{bind-attr src=imageSet.mainImage.url}} width='200' height='200'>
+  {{/if}}
+</div>
+
+<div class='col-sm-2'>
+  {{#if isEditing}}
+    <ul>
+      <li>{{view 'select'
+        content=organizations
```

Comment:
this is the part that may be affected by not blocking to wait for the organizations async call to finish. I am not clear on what happens when the ajax for loading organizations is still in-flight when this part is rendered.